### ISSUE TITLE
Property Editor: Textbox - A11Y

### ISSUE THREAD

**Issue by iOvergaard** [umbraco/Umbraco.CMS.Backoffice#454]
Check the property editor for accessibility issues.

**Comment by portfolioris** [comment]
The contrast of the border is too low. I should have a contrast ratio of 3:1. See https://www.w3.org/TR/WCAG21/#non-text-contrast
